documentation/release-notes-2-8.md

```diff
@@ -1,26 +1,23 @@
 ---
 layout: default
-title: Relax-and-Recover Release Notes
+title: Release Notes for Relax-and-Recover Version 2.8
 ---
 
+This document contains the latest release notes for the Open Source Software project Relax-and-Recover.
 
-# Release Notes for Relax-and-Recover Version 2.8
+Relax-and-Recover website: [relax-and-recover.org](https://relax-and-recover.org/)
 
-This document contains the latest release notes for the Free and Open Source Software project Relax-and-Recover.
+GitHub project: [github.com/rear](https://github.com/rear)
 
-Relax-and-Recover website: <http://relax-and-recover.org>
+For the release notes of previous Relax-and-Recover versions see the
+Relax-and-Recover website release notes: [relax-and-recover.org/documentation](/documentation)
```

Comment:
Relative links won't work when the release-notes file
is included in the rear/rear Github repository
(i.e. in the actual ReaR files)

Reply:
@schlomo 
will relative URLs in documentation/release-notes-2-8.md
work when it is included "as is" in our rear/rear
GitHub repository?

Do relative URLs in documentation/release-notes-2-8.md
perhaps work when release-notes-2-8.md is converted
into ASCII text with Lynx, cf.
https://github.com/rear/rear.github.com/pull/22/files#r1883931162

I.e. will Lynx replace relative URLs by absolute URLs?

Reply:
Yes, relative URLs are translated to full URLs, which is why it is important to dump the release notes from our website and not the local development server:
<img width="623" alt="image" src="https://github.com/user-attachments/assets/37a348a8-f145-4155-a1a4-1f3d91783465" />